bears/general/KeywordBear.py

```diff
@@ -79,22 +79,55 @@ def run(self,
             filename,
             file,
             keywords: list=['todo', 'fixme'],
+            regex_keyword: str='',
             dependency_results: dict=None):
         '''
         Checks the code files for given keywords.
 
         :param keywords:
             A list of keywords to search for (case insensitive).
             Default are TODO and FIXME.
+        :param regex_keyword:
+            A regular expression to search for matching keywords in a file.
         '''
         keywords_regex = re.compile(
             '(' + '|'.join(re.escape(key) for key in keywords) + ')',
             re.IGNORECASE)
 
+        regex = re.compile(regex_keyword)
+
         comments = _get_comments(dependency_results)
 
+        message = "The line contains the keyword '{}'."
+        yield from self.check_keywords(filename, file, comments,
+                                       keywords_regex, message)
```

Comment:
IMO there should be an if around this one as well, the keywords setting may be given empty as well